skill-recorder.lic

```diff
@@ -17,7 +17,7 @@ loop do
   pause 2
   clear
   UserVars.athletics = DRSkill.getrank('Athletics')
-  if DRSpells.active_spells['Athleticism']
+  if DRSpells.active_spells['Athleticism'] || DRSpells.active_spells['Khri Flight'] || DRSpells.active_spells['Unyielding']
     UserVars.athletics = UserVars.athletics * 1.1
```

Comment:
what do you think about making this 15% instead of 10%?

Reply:
Since we don't know how much mana was put into Athleticism, 10% seems safer.

Reply:
yeah 10% makes the most sense since it's the bare minimum. I'll be honest, I didn't even know having a buff up would effect your climbing in ;go2 until I saw this script today and the only levels where I would have noticed it I had 15% at least which is why I was asking. (segoltha and undergondola)